sdk/python/tests/compiler/testdata/artifact_location.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp import dsl
+from kubernetes.client.models import V1SecretKeySelector
+
+
+@dsl.pipeline(name='artifact-location-pipeine', description='hello world')
+def foo_pipeline(tag: str, namespace: str = "kubeflow", bucket: str = "foobar"):
```

Comment:
what does this pipeline actually do? it's hard to tell what should be expected when I run this pipeline, or, what we can look for when we add actual integrations tests for this